sda-commons-server-dropwizard/src/main/java/org/sdase/commons/server/dropwizard/bundles/SystemPropertyAndEnvironmentLookup.java

```diff
@@ -1,19 +1,56 @@
 package org.sdase.commons.server.dropwizard.bundles;
 
-import io.dropwizard.configuration.EnvironmentVariableLookup;
+import com.fasterxml.jackson.core.io.CharTypes;
+import java.util.Optional;
+import org.apache.commons.text.lookup.StringLookup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Lookup for Java's system properties and environment variables. System properties have higher
  * priority.
  */
-public class SystemPropertyAndEnvironmentLookup extends EnvironmentVariableLookup {
+public class SystemPropertyAndEnvironmentLookup implements StringLookup {
```

Comment:
Keep the name although we also apply operators now?
What about short `SdaLookup` and good documentation?

Reply:
I would not rename it because that would cause a breaking change for people who use it without the bundle. However, I can extend the documentation.

Reply:
[Docs added](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2088/commits/f924d37be6d0d2a10447caadd0459498ad0e7608)